src/components/common/AddCityBar.js

```diff
@@ -1,9 +1,17 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import './styles/AddCityBar.scss';
 import AutoCompleteInput from './AutoComplete';
-
+import { ReportContext } from '../../state/contexts/ReportContext';
 const AddCityBar = () => {
   // Search bar for adding cities
-  return <AutoCompleteInput />;
+  let { compareList } = useContext(ReportContext);
```

Comment:
When we finalize city report functions, we should come back to this and see if new tests work with 1 object (i.e. {compareList, setCompareList} = useContext